tests/test_fmi_estimate.py

```diff
@@ -101,11 +97,11 @@ def f(*args, **kwargs):
             model.reset()
 
             # Set optimal values for a1 and a2 into the model
-            model.set('qt.a1',res["qt.a1"])
-            model.set('qt.a2',res["qt.a2"])
+            model.set(['qt.a1'], res["qt.a1"])
```

Comment:
Simply used incorrectly, but was accepted in the DummyModel anyways and broke array concatination in various places, since new Cython is more strict on this.